### PR TITLE
Escape $ to prevent MathJax formatting on currency

### LINKED ISSUE #13335: MathJax formatting being applied to currency examples
The docs page is injecting MathJax for formula formatting but is using only a single $ as an inline delimiter. This is causing the currency formatting examples to format incorrectly.

Suggested fix: Escape at least one of the $ chars.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0ae38fce-fc98-9420-6b0b-ceee4e885f2d
* Version Independent ID: f23035e0-f271-b48c-f9a9-8582be80b4ad
* Content: [Standard Numeric Format Strings](https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-numeric-format-strings#feedback)
* Content Source: [docs/standard/base-types/standard-numeric-format-strings.md](https://github.com/dotnet/docs/blob/master/docs/standard/base-types/standard-numeric-format-strings.md)
* Product: **dotnet**
* Technology: **dotnet-standard**
* GitHub Login: @rpetrusha
* Microsoft Alias: **ronpet**

### PR DESCRIPTION
**Pull request by ChrisRomp**
## Summary

Escape $ in currency example to prevent MathJax formatting on currency #13335.  I'm unsure how to preview this since the MathJax script is being inserted elsewhere.

Fixes #13335

### CODE REVIEW

**Comment by rpetrusha** [comment]
Thanks for contributing to the dotnet/docs repo and pointing out the loss of formatting, @ChrisRomp. Unfortunately, escaping `$` didn't fix the problem. The preview is available only in our internal build. It appears as follows:

image

I've tried using an ASCII escape code (%24), which I'm hoping will be successful.

**Comment by rpetrusha** [comment]
Unfortunately, that didn't work either:

image
I'm trying an HTML code.

**Comment by ChrisRomp** [comment]
I wonder why the MathJax isn't configured to use the standard `$$` inline delimiter... :-/

**Comment by rpetrusha** [comment]
I tried $dollar;, which didn't work and recreates the original issue. I just tried $$.

**Comment by ChrisRomp** [comment]
I was hoping `\$` would work per the docs: http://docs.mathjax.org/en/latest/options/preprocessors/tex2jax.html

Wonder if the markdown to HTML conversion is making some changes...?  No idea, really.

**Comment by rpetrusha** [comment]
I have no idea either. $$ didn't work, so I've tried fencing the en-US result.

**Review by rpetrusha — APPROVED** [review]
This worked and looks OK: 

image

I'll merge your PR, @ChrisRomp. It should be live on docs.microsoft.com sometime tomorrow. Again, thanks for pointing this out.

**Comment by ChrisRomp** [comment]
Beautiful. Thanks. And I'm glad docs is on Github now, so when I see something I can try to do something about it. :)
